examples/simple4.yml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: MIT
+---
+- name: "Apply a multi-file sudoers configuration"
```

Comment:
Remove quotes around strings everywhere here too

Reply:
Can you make names of examples clearer? Something that describes the example content